### ISSUE TITLE
@token is ']

### ISSUE THREAD

**Issue by sweebee** [pimatic/pimatic#820]
Is there a way to disable / remove this error? Its very annoying when editing rules with ```[ ]```

```
19:34:22.577 [pimatic] error: @token is ']'
```

**Comment by sweetpi** [comment]
Could you give an example rule that leads to this error?

**Comment by sweebee** [comment]
All the rules that use '[ ]'. 

Ex:
```
If [[light of woonkamer-ldr <= 85% or its after sunset or its before sunrise] and trigger: PIR overloop is present and overloop-lamp is turned off] or [trigger: PIR overloop is absent and overloop-lamp is turned on] then toggle overloop-lamp
```

**Comment by sweetpi** [comment]
Thanks for reporting, is fixed in pimatic v0.8.102.
